plugins/firstorder/g_ground.ml4

```diff
@@ -40,17 +40,17 @@ let _=
   in
     declare_int_option gdopt
 
-let congruence_depth=ref 100
 
 let _=
+  let congruence_depth=ref 100
   let gdopt=
-    { optdepr=false;
+    { optdepr=true; (* noop *)
```

Comment:
I'm not sure why this is in a documentation PR - shouldn't deprecating options be separate from documenting them?